wa/instruments/proc_stat/gather-load.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
This shebang causes the script to fail on devices that do not have sh present at that path. Can this be updated to invoke via the busybox implementation?

Reply:
Yup, done.